libswirl/rend/gles/imgui_impl_opengl3.cpp

```diff
@@ -134,13 +134,15 @@ void    ImGui_ImplOpenGL3_RenderDrawData(ImDrawData* draw_data, bool save_backgr
     // Backup GL state
     glActiveTexture(GL_TEXTURE0);
     bool clip_origin_lower_left = true;
-
+    
+#if HOST_OS != OS_DARWIN
```

Comment:
This is feels like a bit of a hack :/

Reply:
I'll take another look at this one. I don't remember why I did it that way.

Reply:
So the problem here is that `glClipControl` and `GL_CLIP_ORIGIN` are undeclared identifiers on Mac, so I had used the #if block to remove them from the Mac build entirely.

Reply:
That looks like a bug in our headers/glwrap. Can you add a comment about it?